### DISCUSSION TITLE
Turn on/off notification (Ring Intercom)

### DISCUSSION THREAD

**Discussion by developerGM** [tsightler/ring-mqtt#686] [General Discussion]
Hi,
from Ring app I can turn on/off, using a switch, the notification when someone ring on my door.

How can I do using this addon?
I see that I can only lock/unlock the door and check some sensors info

**Comment by tsightler** [comment]
Available features are listed in the documentation, if a feature is not listed there then it isn't available.  I don't own an intercom, so I don't know much about it, but assuming it works similar to a doorbell, it doesn't really make sense to enable/disable notifications via ring-mqtt as those settings are device specific (i.e. if you have mulitple devices with the Ring app the notificaitons are only disabled for the specific device on which they are turned on/off).

**Reply by developerGM** [reply]
I think that this can make sense because I can create an automation in ha that during the night don't send alert notification with sound to our mobile phones
I disable sound from my original intercom so the sound does not wake sleeping babies

From the Ring app you can do it
![Senza titolo](https://github.com/tsightler/ring-mqtt/assets/9252653/d1cc19d5-c9fa-4613-a80e-df360c419e2d)

**Reply by developerGM** [reply]
this is in Italian language
I am talking about the switch ->  "Avvisi di chiamata"

**Reply by tsightler** [reply]
>  I can create an automation in ha that during the night don't send alert notification with sound to our mobile phones

But that's exactly what I said above won't work.  I'll try to explain more clearly.

Notifications are a device/app level setting, if you have two phones with the Ring app installed, disabling "Avvisi di chiamata" on phone1 will not disable the setting for phone2.  As the ring-mqtt process is basically just another phone, it can only enable/disable "Avvisi di chiamata" for itself, it can't change that setting for your phone.

**Reply by developerGM** [reply]
yes, I understand very well what you said...I am an android developer so I know that app normally set notification phone by phone so if I turn off notification in phone1...I don't have turn off on phone2

Reading settings I found `'subscriptions': {'event_types': []}`...for this I asked if this json object can manage these events.

that's it
Thanks for you time

**Reply by tsightler** [reply]
It can, but for individual devices.  If you use something like mitmproxy or the Ring web dashboard and developer tools, you will see that there are API calls to subscribe/unsubscribe from events when you toggle these settings, however, those API calls affect only the device from which they are called.

Theoretically it could be possible to query the device IDs of those other devices and fake calls to the API as those devices, or perhaps somehow call the API to unsubscribe other device as it appears to just be based on the hardware ID included in the request header, but that is far outside the scope of ring-mqtt.
